keps/sig-auth/3766-referential-authorization/README.md

```diff
@@ -558,52 +568,8 @@ moving ReferenceGrant to the new API group, just notes to save discussion time.
   specific `To` Kinds.
 * Corollary for future work, define how controllers interact. Is it a problem if
   multiple controllers consume the same ReferenceGrant?
-* Status design is still pending, but it's currently expected that controllers
-  will indicate status on the _referring_ resources, not on ReferenceGrant itself.
-<<[UNRESOLVED do we need status? ]>>
-Original Thread: https://github.com/kubernetes/enhancements/pull/3767#discussion_r1084670421
```

Comment:
Could we leave a little bit of explanation of this, why we don't want to put it on the ReferenceGrant side?

Reply:
Added a note below in "Do We Need Status?" section that replaced this, the key point is that it's difficult to handle status here because it's unclear who would populate it (many controllers involved).